mm-go-irckit/server.go

```diff
@@ -444,6 +444,21 @@ outerloop:
 				u.Pass = msg.Params
 			case irc.JOIN:
 				s.EncodeMessage(u, irc.ERR_NOTREGISTERED, []string{"*"}, "Please register first")
+			// https://ircv3.net/specs/extensions/capability-negotiation.html
+			case irc.CAP:
+				subcommand := msg.Params[0]
+				switch subcommand {
+				case irc.CAP_LS, irc.CAP_LIST:
+					params := fmt.Sprintf("* %s :", subcommand)
+					s.EncodeMessage(u, irc.CAP, []string{params}, "")
```

Comment:
@42wim not sure whether to `//nolint:errcheck` these to silence the golangci-lint failures or actually check and discard the return values. Both of which would make it inconsistent with the rest of the s.EncodeMessage() calls within this file.

Reply:
fine to //nolint it

Reply:
you need to set //nolint:errcheck because of changes in go1.19 wrt gofmt

Reply:
Pushed up a fix, thanks!